can-control.js

```diff
@@ -13,9 +13,9 @@ var assign = require("can-util/js/assign/assign");
 var isFunction = require("can-util/js/is-function/is-function");
 var each = require("can-util/js/each/each");
 var dev = require("can-util/js/dev/dev");
+var vDom = require("can-util/dom/document/document");
```

Comment:
this probably shouldn't be called `vDOM`.  There's nothing `v` about it.  All this does is return the document.